docs/zh/command/cinfo.md

```
@@ -18,11 +18,23 @@ cinfo
-  - alloc： 节点已被占用
-  - down： 节点不可用
+- **STATE**： 节点状态，包括资源状态和控制状态
+  -   **资源状态**
+    -     idle： 空闲
+    -     mix： 节点部分核心可以使用
+    -     alloc： 节点已被占用
+    -     down： 节点不可用
+  -   **控制状态**
+    -     power_idle：空闲态。被调度睡眠时计入power_to_sleeping；有作业则进入power_activate
+    -     power_activate：活跃态，初始状态。没有作业等待一段时间后，部分节点会进入power_idle
+    -     power_sleeping：睡眠态。被调度wakeup时进入power_waking_up;被调度挂机后，进入power_powering_off
```

markdownlint-cli2 (0.21.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 4

(MD005, list-indent)